src/main/sensors/gyro.c

```diff
@@ -311,10 +312,23 @@ void gyroInitFilters(void)
 #endif
 
     if (gyroConfig()->gyro_soft_lpf_hz) {
-        softLpfFilterApplyFn = (filterApplyFnPtr)biquadFilterApply;
-        for (int axis = 0; axis < 3; axis++) {
-            softLpfFilter[axis] = &gyroFilterLPF[axis];
-            biquadFilterInitLPF(softLpfFilter[axis], gyroConfig()->gyro_soft_lpf_hz, getLooptime());
+        
+        switch (gyroConfig()->gyro_soft_lpf_type) 
+        {
+        case FILTER_PT1:
+            softLpfFilterApplyFn = (filterApplyFnPtr)pt1FilterApply;
+            for (int axis = 0; axis < 3; axis++) {
+                softLpfFilter[axis] = &gyroFilterLPF[axis];
+                pt1FilterInit(softLpfFilter[axis], gyroConfig()->gyro_soft_lpf_hz, getLooptime());
```

Comment:
I think it should be `getLooptime() * 1e-6f`

Reply:
Thanks!